ansible_ai_connect/main/settings/base.py

```diff
@@ -24,63 +24,77 @@
 https://docs.djangoproject.com/en/4.1/ref/settings/
 """
 
+import logging
 import os
 import sys
 from importlib.resources import files
 from pathlib import Path
 from typing import Literal, cast
 
+logger = logging.getLogger(__name__)
+
 BASE_DIR: Path = files("ansible_ai_connect")
 ANSIBLE_AI_PROJECT_NAME = os.getenv("ANSIBLE_AI_PROJECT_NAME") or "Ansible AI Connect"
 
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
 
-ANSIBLE_AI_MODEL_MESH_HOST = os.getenv(
-    "ANSIBLE_AI_MODEL_MESH_HOST", "https://model.wisdom.testing.ansible.com"
-)
-ANSIBLE_AI_MODEL_MESH_INFERENCE_PORT = os.getenv("ANSIBLE_AI_MODEL_MESH_INFERENCE_PORT", 443)
-ANSIBLE_AI_MODEL_MESH_INFERENCE_URL = (
-    f"{ANSIBLE_AI_MODEL_MESH_HOST}:{ANSIBLE_AI_MODEL_MESH_INFERENCE_PORT}"
-)
-
+# ==========================================
+# Model Provider
+# ------------------------------------------
 t_model_mesh_api_type = Literal[
     "grpc", "http", "dummy", "wca", "wca-onprem", "wca-dummy", "ollama", "llamacpp", "bam"
 ]
 ANSIBLE_AI_MODEL_MESH_API_TYPE: t_model_mesh_api_type = os.getenv(
     "ANSIBLE_AI_MODEL_MESH_API_TYPE"
 ) or cast(t_model_mesh_api_type, "http")
-t_model_mesh_healthcheck_protocol = Literal["http", "https"]
-ANSIBLE_AI_MODEL_MESH_API_HEALTHCHECK_PROTOCOL: t_model_mesh_healthcheck_protocol = os.getenv(
-    "ANSIBLE_AI_MODEL_MESH_API_HEALTHCHECK_PROTOCOL"
-) or cast(t_model_mesh_healthcheck_protocol, "https")
-ANSIBLE_AI_MODEL_MESH_API_HEALTHCHECK_PORT = (
-    ANSIBLE_AI_MODEL_MESH_INFERENCE_PORT
-    if ANSIBLE_AI_MODEL_MESH_API_TYPE == "http"
-    else (
-        os.getenv("ANSIBLE_AI_MODEL_MESH_API_HEALTHCHECK_PORT", "8443")
-        if ANSIBLE_AI_MODEL_MESH_API_TYPE == "grpc"
-        else None
-    )
-)
-
-ANSIBLE_AI_MODEL_NAME = os.getenv("ANSIBLE_AI_MODEL_NAME") or "wisdom"
 
+t_model_mesh_protocol = Literal["http", "https"]
+ANSIBLE_AI_MODEL_MESH_PROTOCOL: t_model_mesh_protocol = os.getenv(
+    "ANSIBLE_AI_MODEL_MESH_PROTOCOL"
+) or cast(t_model_mesh_protocol, "https")
+ANSIBLE_AI_MODEL_MESH_HOST = (
+    os.getenv("ANSIBLE_AI_MODEL_MESH_HOST") or "https://model.wisdom.testing.ansible.com"
+)
+ANSIBLE_AI_MODEL_MESH_INFERENCE_PORT = int(
+    os.getenv("ANSIBLE_AI_MODEL_MESH_INFERENCE_PORT") or "443"
+)
+ANSIBLE_AI_MODEL_MESH_INFERENCE_URL = os.getenv("ANSIBLE_AI_MODEL_MESH_INFERENCE_URL") or (
+    f"{ANSIBLE_AI_MODEL_MESH_PROTOCOL}://"
```

Comment:
Note the defaults don't make sense anymore since we've added in `ANSIBLE_AI_MODEL_MESH_PROTOCOL` since the protocol was already included in `ANSIBLE_AI_MODEL_MESH_HOST`.

Reply:
Note that using the individual env vars for grpc will no longer work, since in that case the full inference URL does not take a protocol.

Reply:
It's better IMO to only set `ANSIBLE_AI_MODEL_MESH_INFERENCE_URL` and avoid relying on the dynamic variable construction.